e2e/application/application_test.go

```diff
@@ -128,14 +128,6 @@ var ApplicationInitIntercativeCliContext = func(context string, appName string,
 				data := []struct {
 					q, a string
 				}{
-					{
-						q: "What is the domain of your application service (optional): ",
-						a: "testdomain",
-					},
-					{
-						q: "What is your email (optional, used to generate certification): ",
-						a: "test@mail",
```

Comment:
can we keep the domain and email as metadata?

Reply:
Yes, but IMO this can be confusing because we didn't acturally use it.